code/game/objects/structures/disaster_counter.dm

```diff
@@ -0,0 +1,75 @@
+/**
+  * # Disaster counter.
+  *
+  * Tracks how many shifts it has been since the counter with that ID was exploded.
+  */
+/obj/structure/disaster_counter
+	name = "disaster counter"
+	desc = "This magical device will count up how many shifts it has been since a major disaster in this area."
```

Comment:
```suggestion
	desc = "This device how many shifts it has been since a major disaster in this area. A safe workplace is a happy workplace."
```

Reply:
Will do

Reply:
Changed to `"This device will counts how many shifts it has been since a major disaster in this area. A safe workplace is a productive workplace."`, because NT cares more about productivity than employee happiness

Reply:
"Will counts"?